types/usb/index.d.ts

```diff
@@ -382,6 +382,12 @@ export interface Endpoint extends EventEmitter {
 
   /** Object with fields from the endpoint descriptor -- see libusb documentation or USB spec. */
   descriptor: EndpointDescriptor;
+
+  /**
+   * Clear the halt/stall condition for this endpoint.
+   * @param callback
+   */
+  clearHalt(callback: (error?: string) => void): void;
 }
```

Comment:
Others areas call the callback method `cb`, I'd prefer to see this consistent or changed everywhere.

Reply:
Also, should it be optional?